src/features/dataset/state/datasetActions.ts

```diff
@@ -36,7 +36,7 @@ function fetchDataset (ref: QriRef): ApiAction {
     type: 'dataset',
     ref,
     [CALL_API]: {
-      endpoint: 'dataset_summary',
+      endpoint: 'ds/get',
```

Comment:
`ds/get` exists & works the same on core and cloud and should be used instead.

Reply:
ageree. I think we should go so far as to remove `dataset_summary` and mandate that `ds/get` without a selector returns a preview